source/_integrations/air_quality.markdown

```diff
@@ -2,6 +2,7 @@
 title: Air Quality
 description: Instructions on how to add air quality sensors with Home Assistant
 ha_release: 0.85
+ha_iot_class: Cloud Polling
```

Comment:
This is a component (in the old terminology) and does not do anything. The platforms are responsible to interact with the devices/devices/etc.